src/System.Collections.Concurrent/tests/System.Collections.Concurrent.Tests.builds

```diff
@@ -8,6 +8,9 @@
       <TestTFMs>netcoreapp1.0</TestTFMs>
     </Project>
     <Project Include="System.Collections.Concurrent.Tests.csproj">
+      <TargetGroup>netcoreapp1.1</TargetGroup>
```

Comment:
I had that originally, but removed based on feedback from Jose as it's the default. https://github.com/dotnet/corefx/pull/13828#discussion_r88946174